docs/pages/index.mdx

```diff
@@ -2,39 +2,134 @@ import ImageRow from "@/components/ImageRow";
+
+Pelican serves two sides of the same coin -- Data owners who want to federate their data from wherever it lives natively, and data consumers who want to access and compute on data wherever they need it.
+
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-repo-and-consumer.png"}>
+	The federation's core goal is connecting data owners and data consumers.
+</ImageRow>
+
+
+As such, the primary prerequisite for data to be moved via a Pelican federation is for a data owner to make their data accessible to the federation. This happens when an Origin is placed in front of the repository and registered with the federation. While federations like the OSDF *may* wish to control or filter any Origin registrations to vet the data they make available, this example assumes the Origin's registration is automatically approved. The red arrow in the following graphic represents the vetting/approval step, should the federation require it.
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-origin-registration.png"}>
+	When the Origin starts, it's told which federation it belongs to. It discovers the
+	hostnames for its Registry and Directory by using the federation root to construct
+	the URL "https://osg-htc.org/.well-known/pelican-configuration", which contains a JSON
+	detailing the federation's central services.
+
+	Next, the Origin registers its namespace and public key with the Registry, proving that it
+	owns the corresponding private key. Finally, the Origin begins advertising its namespace
+	information and hostname to the Director.
+</ImageRow>
+
+While somewhat simplified, this example illustrates the process origins must take to make themselves known within the federation. After completing these steps, the objects from the Data Repository are available via Pelican.
+
+The next step is for the data consumer to actually *move* the data. Pelican assumes the data consumer already knows the federation that provides the data they want, along with the name of the object within the federation. These two pieces of information are combined and provided to the Client as a `pelican://`-schemed URL
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-orig-discovery.png"}>
+	The data consumer provides their Pelican client of choice the pelican:// URL that defines
+	the object they want to download, where osg-htc.org is the federation and
+	(/weather/cloud.jpg) is the object. Just as the origin discovered the Director's hostname
+	by visiting "https://osg-htc.org/.well-known/pelican-configuration", so too does the client.
+
+	After the client has performed federation metadata discovery, it issues an HTTP GET
+	request to the Director, using the object name as a URL path. The Director responds
+	with an HTTP 307 Redirect, forwarding the client on to the a server that can provide
+	the object, in this example an Origin.
+
+	Finally, the Client follows the redirect and downloads the object by issuing an HTTP
```

Comment:
Is there a specific reason that we skip the staging components? I'm asking because I don't think by default origin (or director) allows direct access to the origin

Reply:
Technically neither of those has an enforcement mechanism -- If I ask the director for an origin redirect, it gives it to me, and if some client shows up to the origin with a valid token, the origin doesn't ask whether the client is a cache or something else.